packages/angular-workspace/package.json

```diff
@@ -61,9 +61,9 @@
     "karma-jasmine": "^5.1.0",
     "karma-jasmine-html-reporter": "^2.0.0",
     "karma-spec-reporter": "^0.0.36",
-    "ng-packagr": "^16.2.3",
+    "ng-packagr": "^17.3.0",
     "playwright": "1.44.0",
     "rollup": "^4.12.0",
-    "typescript": "~5.1.6"
+    "typescript": "~5.4.5"
```

Comment:
The angular support table doesn't says <5.4 is supported: https://v17.angular.io/guide/versions#actively-supported-versions, so maybe all these should be `"typescript": "~5.3.3"`

Reply:
Ah dang, was looking at the wrong site, this one is okay: https://angular.dev/reference/versions